Compiler/src/main/java/com/adguard/compiler/Browser.java

```diff
@@ -22,19 +22,14 @@
 	SAFARI("safari"),
 
 	/**
-	 * Firefox browser
+	 * Firefox browser (legacy XPCOM)
 	 */
-	FIREFOX("firefox"),
+	FIREFOX_LEGACY("firefox"),
```

Comment:
Yeap, it doesn't exists, but for a long time, then we see "legacy" we will think about the old one legacy.